src/frame.rs

```diff
@@ -101,11 +102,13 @@ impl<'a> Frame<'a> {
             Value::Float(f) => match op.operation {
                 OpName::Plus => Rc::new(Value::Float(f)),
                 OpName::Minus => Rc::new(Value::Float(-f)),
+                OpName::Multiply | OpName::Divide => panic!("Bad unary operation"),
```

Comment:
```suggestion
                OpName::Multiply | OpName::Divide => panic!("Cannot apply the unary operation {:#?} to the value {} of type Float", op.operation, f),
```

I have local code that does similar to this for the unary errors but not a chance to form into a PR yet